google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -204,6 +205,14 @@ public static BucketTargetOption metagenerationNotMatch() {
     public static BucketTargetOption userProject(String userProject) {
       return new BucketTargetOption(StorageRpc.Option.USER_PROJECT, userProject);
     }
+
+    /**
+     * Returns an option to define the projection in the API request. In some cases this option may be needed to be
+     * set to `noAcl` to omit ACL data from the response.
```

Comment:
State the default value is "full" as well. Add the reference link as well. https://cloud.google.com/storage/docs/json_api/v1/buckets/patch